.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - "6"
+  - "8"
```

Comment:
Should Node 4 also be added here? There are still a lot of people on 4, and most of our other modules maintain compatibility with it.

Reply:
ping @horia-delicoti ^

Reply:
The tests were failing on node 4: https://travis-ci.org/resin-io/node-lkl/jobs/290434898
I needed this merged for node 8 support.
We can add node 4 support in a new PR.